src/common/components/Tooltip/Skin/index.js

```diff
@@ -12,17 +12,8 @@ type Props = {
 const SkinTooltip = ({ data: { skin } }: Props) => {
   return (
     <Background>
-      <div className={styles.itemHeader}>
-        <Icon size="mini" src={skin.icon} className={styles.tooltipIcon} />
-        <span className={styles.skinNameText}>
-          {skin.name}
-        </span>
-      </div>
-      <div>
-        <span className={styles.skinTypeText}>
-          {skin.details.type}
-        </span>
-      </div>
+      <ItemHeader name={skin.name} icon={skin.icon} rarity={'white'} />
+      <span className={styles.skinTypeText}>{skin.details.type}</span>
```

Comment:
for consistency with the ingame tooltip can we have the item weight and the item type displayed?

armor:
```
https://api.guildwars2.com/v2/skins/1028
skin.details.weight_class
`${skin.details.type}${skin.type}`
```

weapons:
```
https://api.guildwars2.com/v2/skins/7756
skin.details.type
```

and maybe add a test or two if you can so the code coverage goes green? :-)

Reply:
Just so its clear, 

For armor: we'd want to show Heavy Armor or Heavy Helm? (helm being more specific)
For weapons: we'd want to show Hammer vs. Weapon.